Sources/Testing/Testing.docc/BugIdentifiers.md

```diff
@@ -44,14 +47,15 @@ handling to detect:
 
 ## Examples
 
-| Trait Function | Inferred Bug-Tracking System |
-|-|-|
-| `.bug(12345)` | None |
-| `.bug("12345")` | None |
-| `.bug("rdar:12345")` | Apple Radar |
-| `.bug("https://github.com/apple/swift/pull/12345")` | [GitHub Issues for the Swift project](https://github.com/apple/swift/issues) |
-| `.bug("https://bugs.webkit.org/show_bug.cgi?id=12345")` | [WebKit Bugzilla](https://bugs.webkit.org/) |
+| Trait Function | Valid | Inferred Bug-Tracking System |
+|-|:-:|-|
+| `.bug(12345)` | Yes | None |
+| `.bug("12345")` | Yes | None |
+| `.bug("Things don't work")` | **No** | None |
+| `.bug("rdar:12345")` | Yes | Apple Radar |
+| `.bug("https://github.com/apple/swift/pull/12345")` | Yes | [GitHub Issues for the Swift project](https://github.com/apple/swift/issues) |
+| `.bug("https://bugs.webkit.org/show_bug.cgi?id=12345")` | Yes | [WebKit Bugzilla](https://bugs.webkit.org/) |
 <!--
-| `.bug("FB12345")` | Apple Feedback Assistant | // SEE ALSO: rdar://104582015
-| `.bug("#12345")` | GitHub Issues for the current repository (if hosted there) |
+| `.bug("FB12345")` | Yes | Apple Feedback Assistant | // SEE ALSO: rdar://104582015
```

Comment:
I'm a little confused why this is still considered valid. Does it parse as an integer?

Reply:
It's commented out as a hypothetical "future direction."